configure

```diff
@@ -396,3 +394,16 @@ add_var HAVE_PYTHON3_MAKO "$HAVE_PYTHON3_MAKO"
 # Hack to avoid sha256 name clash with libwally: will be fixed when that
 # becomes a standalone shared lib.
 echo '#include "ccan_compat.h"' >> $CONFIG_HEADER
+
+# Now we set them all and check.
+while IFS='=' read VAR VAL; do
+    eval $VAR=\"$VAL\"
+done < $CONFIG_VAR_FILE.$$
+
+if [ "$HAVE_SQLITE3" = 0 -a "$HAVE_POSTGRES" = 0 ]; then
+    # I have no database yet I must schema!)
+    echo "*** We need a database, but neither sqlite3 nor postgres found" >&2
+    exit 1
```

Comment:
Exiting here would leave the temporary file `$CONFIG_VAR_FILE.$$` in place. Is that what you want?

Reply:
Oops, I assumed there was a "trap" to clean this up (there are other cases we can exit early, due to `set -e`).  Will fix that separately...